data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1179 @@
+# Copyright (c) 2023 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+mission "Passive-Aggressive Observations"
+#	invisible
+	landing
+	destination "Vigales"
+	to offer
+#		random < 30
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on complete
+		"gegno: bgs" ++
```

Comment:
maybe this could be made a bit more explicit?

Reply:
I'm not sure how to make "bgs (background simulator)" more explicit. Besides, who is it supposed to be more explicit for? People editing their save? This is a very particular mission series, which shouldn't be edited really, and if it does, "bgs" is a simple and recognizable enough term to label these events.

Reply:
I was mostly thinking for ppl reading the mission files trying to understand it haha
"gegno: encounters" would already be clearer

Reply:
Fair enough.